### PR TITLE
Add package for py2app.

### PR DESCRIPTION
**Pull request by pr0d1r2**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Please fix all the `brew audit --strict py2app` warnings: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26743/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_py2app___strict/

File: Library/Formula/py2app.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://pypi.python.org/packages/source/p/py2app/py2app-0.9.tar.gz"
+  sha256 '7922672f9e99b50ed931780d43405ac134988b1532dd0659ef130b824f88c59d'
+
+  version "0.9"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", "#{lib}/python2.7/site-packages"
+    system "python", "setup.py", "install", "--prefix=#{prefix}"
```
Comment (mikemcquaid) [inline Library/Formula/py2app.rb]: Is this a library or a binary?

Comment (pr0d1r2) [inline Library/Formula/py2app.rb]: This is binary used for creating OSX apps from python scripts.

**Comment by DomT4** [comment]
I'd recommend looking at [jrnl](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/jrnl.rb#L42-L66) for how Python installations should be handled. You'll need to vendor `altgraph`, `modulegraph` and `macholib` which are all dependencies.

**Comment by pr0d1r2** [comment]
Pass `brew audit --strict py2app.rb`

**Comment by pr0d1r2** [comment]
@DomT4 dependencies included.

File: Library/Formula/py2app.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system("touch example.py")
```
Comment (DomT4) [inline Library/Formula/py2app.rb]: You don't need to call the system here. You can just do `touch "example.py"`.

File: Library/Formula/py2app.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    system("py2applet --make-setup example.py")
```
Comment (DomT4) [inline Library/Formula/py2app.rb]: This should be `system "py2applet", "--make-setup", "example.py"`

**Comment by pr0d1r2** [comment]
@DomT4 Done :)

**Comment by DomT4** [comment]
Thanks :tada:, looks great as far as I can see.

**Comment by pr0d1r2** [comment]
Achievement unlocked: Homebrew Contributor ;)

**Comment by tdsmith** [comment]
Installing py2app this way, it won't work as a setuptools command; it'll only work when it's invoked as `py2applet`, which may not be what people expect. What's the advantage of installing this with Homebrew and not with pip? (There's no blanket prohibition on Homebrew packaging things that pip also packages, but sometimes Homebrew doesn't seem like a good fit.)

**Comment by pr0d1r2** [comment]
@tdsmith my initial intention was to provide `py2applet` command to experiment with building Mac OS X apps without polluting Cellar with pip installments. Perhaps changing name of package to `py2applet` will be a good idea?

**Comment by tdsmith** [comment]
Passing on this, sorry; I don't think a systemwide installation of the py2app script is useful.

**Pull request closed without merging**